src/app/components/app-builds/app-builds.component.ts

```diff
@@ -92,9 +93,26 @@ export class AppBuildsComponent implements OnInit, OnDestroy {
           this.update();
         }
       });
+
+    this.subUpdate = this.socketService.outputEvents
+      .filter(event => event.data === 'build restarted' || event.data === 'build succeeded'
+        || event.data === 'build failed')
+      .subscribe(event => {
+        let index = this.builds.findIndex(i => i.id === event.build_id);
+        if (index !== -1) {
+          this.apiService.getBuild(event.build_id, this.userId).subscribe(b => {
```

Comment:
here too, remove the whole subscription that you added. if you need to get new times from the database (not sure why) then send it over via socket.